client/common/sass/components/_blog-card.sass

```diff
@@ -5,9 +5,12 @@
 	&--info
 		margin-top: 12px
 
-	&--teaser
+	&__teaser
 		margin-top: 12px
 		margin-bottom: 12px
 
+		&--sm
```

Comment:
So I just used `sm` as small similar to bootstrap notation. This is basically because I wanted the teaser to appear in smaller screens in blog listing page as mentioned in the blog index page design, but not show them in any of the other blog listing pages. I could have used a separate css, but this was the only change, so I went with it. This is related to https://github.com/freedomofpress/pressfreedomtracker.us/pull/1229/files/26ef4422272de8ac0725f1bca5aaa4805b5407d2#diff-de7a921470ee8b60b88019e99f2aaeb91b280301a6c29ae469f7bc06ae998478R51-R53 and https://github.com/freedomofpress/pressfreedomtracker.us/pull/1229/files/26ef4422272de8ac0725f1bca5aaa4805b5407d2#diff-94a3f3f742eaf4f8c35ee2cff00afb0dbba0e983977f2794cb8835bb92271e49R18-R20